src/v2/Apps/Artist/Routes/Overview/Components/ArtistRelatedGeneCategories.tsx

```diff
@@ -0,0 +1,96 @@
+import { Pill, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ArtistRelatedGeneCategories_artist } from "v2/__generated__/ArtistRelatedGeneCategories_artist.graphql"
+import { ArtistRelatedGeneCategoriesQuery } from "v2/__generated__/ArtistRelatedGeneCategoriesQuery.graphql"
+
+interface ArtistRelatedGeneCategoriesProps {
+  artist: ArtistRelatedGeneCategories_artist
+}
+
+const ArtistRelatedGeneCategories: FC<ArtistRelatedGeneCategoriesProps> = ({
+  artist,
+}) => {
+  const categories = extractNodes(artist.related?.genes)
+
+  if (categories.length === 0) return null
+
+  return (
+    <>
+      <Text variant="lg-display">Related Categories</Text>
+
+      <Spacer mt={4} />
+
+      {categories.map(category => {
+        return (
+          <Pill mr={1} mb={1}>
```

Comment:
I'd cast the `Pill` as a link with `as={RouterLink}` since `Pill` is by default a `button`. You'll therefore have two tabstops, one which doesn't do anything. You'll have to `@ts-ignore` the `as` related stuff (which is fine).

Reply:
fixed